lib/util/series_ext.dart

```diff
@@ -43,8 +43,15 @@ extension FFSeriesExt on FFSeries {
           ? (metadata!['mediumDescription'] as List<dynamic>).join('\n')
           : null;
 
-  String? get thumbnailUrl =>
-      thumbnailURI == null ? null : getFFUrl(thumbnailURI!);
+  String? get thumbnailUrl {
+    final uri = (thumbnailDisplay?.isNotEmpty ?? false)
+        ? thumbnailDisplay
+        : thumbnailURI;
+    if (uri == null || uri.isEmpty) {
```

Comment:
use the same logic with artwork, exhibition thumbnailURL and check null or empty in `getFFUrl` function

Reply:
Just keep current logic. Update getFFUrl might lead to the function return null, and we have update logic in other locations too.